Makefile.inc1

```diff
@@ -3522,6 +3537,21 @@ check-old-files: .PHONY
 			echo $${catpage}; \
 	        fi; \
 	done | sort
+.if ${MK_MANCOMPRESS} != "yes"
```

Comment:
I wonder if this could somehow just be on the inner if rather than having two identical calls copies of the loop

Reply:
> I wonder if this could somehow just be on the inner if rather than having two identical calls copies of the loop

The problem are the backslashes.

Reply:
Let me play with something to overcome that

Reply:
I've not found time to do this, so please don't let it block this. We/I can refine after the comment when I find time.